helm-charts/easegress/templates/PersistentVolume.yaml

```diff
@@ -25,6 +25,12 @@ spec:
 ---
 {{ end }}
 
-{{- include "easegress-lib-chart.storageclass" (list . "easegress.storageclass") -}}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: easegress-storage
+provisioner: kubernetes.io/no-provisioner
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
 
-{{- end }}
+{{ end }} 
```

Comment:
Nit: `{{- end}}` will not print extra empty line while `{{ end }}` does. Ref: [[controlling whitespace](https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace)]